modules/ROOT/pages/configuration/plugins.adoc

```diff
@@ -0,0 +1,59 @@
+[[plugins]]
+= Plugins
+:description: This page describes how to load plugins to a Neo4j deployment.
+
+This page describes how to load plugins to a Neo4j deployment.
+The following plugins are supported:
+
+.Supported Neo4j plugins
+[options="header",cols="d,m,b,a"]
+|===
+|Name |Key  | Licence required | Further information
+
+| APOC
+| `apoc`
+| {cross-mark}
+| https://neo4j.com/docs/apoc/current/[APOC user guide]
+
+| Bloom
+| `bloom`
+| {check-mark}
+| link:{neo4j-docs-base-uri}/bloom-user-guide[Neo4j Bloom]
+
+| Streams
+| `streams`
+| {cross-mark}
+| link:{neo4j-docs-base-uri}/kafka-streams[Neo4j Streaming Data Integrations User Guide]
+
+| Graph Data Science
+| `graph-data-science`
+| {cross-mark}
+| link:{neo4j-docs-base-uri}/graph-data-science[Graph Data Science]
+
+| Neo Semantics
+| `n10s`
+| {cross-mark}
+| https://neo4j.com/labs/nsmtx-rdf/[Neo Semantics]
+|===
+
+[NOTE]
+====
+Refer to these other pages on plugins in case you are using link:https://neo4j.com/docs/desktop-manual/current/operations/install-plugin/[Neo4j Desktop], xref:docker/operations.adoc#docker-neo4j-plugins[Docker], or xref:/kubernetes/plugins/[Kubernetes]. 
+====
+
+== Loading plugins 
+
+Here are the steps to load the plugins:
+
+. Go to `<NEO4J_HOME>/products` and copy or move the chosen plugins (.jar files) into the `<NEO4J_HOME>/plugins` directory.
+See the table in xref:configuration/file-locations/[File locations] for more information.
+
+. *Optional*: Some plugins, such as Bloom, require licencing and configuring to become available to the user within Neo4j.
```

Comment:
I think all of them require conifguration but some also require an activation key.